scripts/networking_setup_pi

```diff
@@ -1,7 +1,7 @@
-export ROS_MASTER_URI=http://10.0.0.2:11311
+
```

Comment:
I think this was probably just a testing change for the pi and should be removed?